apps/pre/pre-api/prod.yaml

```diff
@@ -7,4 +7,6 @@ spec:
   releaseName: pre-api
   values:
     java:
-      ingressHost: pre-api.platform.hmcts.net
\ No newline at end of file
+      ingressHost: pre-api.platform.hmcts.net
+    keyvault:
```

Comment:
can you explain how this will work?

I don't see how this will make this work?
https://github.com/hmcts/pre-api/blob/master/charts/pre-api/values.yaml#L6-L7

Reply:
just updated so the keyvault is declared under java. hoping that this will overwrite the default values declared in that link ^. if it works the other way around, would we need to create a values.prod.template.yaml file ?